lib/config/defaults.js

```diff
@@ -130,7 +130,7 @@ Object.defineProperty(exports, 'defaults', {get: function () {
 
     cidr: null,
 
-    color: true,
+    color: typeof(process.env.NO_COLOR) === 'undefined',
```

Comment:
According to the error on CI, we might need to replace here with this: `typeof process.env.NO_COLOR`.

Refs: https://github.com/eslint/eslint/blob/master/docs/rules/no-extra-parens.md

Reply:
This is better off as `!process.env.NO_COLOR` or, if you want `NO_COLOR=""` to be truthy, `process.env.NO_COLOR != null` (with one `=`)